pom.xml

```diff
@@ -71,7 +72,7 @@
                     <source>1.8</source>
                     <target>1.8</target>
                     <!-- we want to fail if we depend on any deprecated methods -->
-                    <failOnWarning>true</failOnWarning>
+                    <failOnWarning>false</failOnWarning>
```

Comment:
is it not possible to fix the warnings?

Reply:
The warnings are for different reasons including using deprecated methods. Fixing them needs changing the code.
Should not I move the code without any changes and then fix any existing problem so that we have the rack of changes in the original code? I mean they should be fixed in another issue. Including removing the comments?

Reply:
Thanks for the second commit addressing these